Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
+  head "https://svn.cyberduck.io/trunk/"
```

Comment:
Nitpick but as you're changing stuff anyway: put `head` below `sha1` if that's cool, thanks.